### PR TITLE
HL-23

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-23)

- Fixes table order
- Adds filtering for amount of bets in table
- Adds pagination

image
image
image
image
image

### CODE REVIEW

**Review by OptimusOpus — DISMISSED** [review]

**Review by OptimusOpus — DISMISSED** [review]

File: src/pages/Bets/components/BetRows.tsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 type Props = {
+  myBetsSelected: boolean;
   bets?: BetHistory[];
   onClickBet: (bet?: BetHistory) => void;
 };
 
-const BetRows: React.FC<Props> = ({ bets, onClickBet }) => {
+const BetRows: React.FC<Props> = ({ myBetsSelected, bets, onClickBet }) => {
+  const { isConnected } = useAccount();
+  if (!isConnected && myBetsSelected)
```
Comment (xiangzhe0812) [inline src/pages/Bets/components/BetRows.tsx:14]:
```suggestion
  if (!isConnected)
```

Comment (brettvniekerk) [reply src/pages/Bets/components/BetRows.tsx:14]: the user can be not connect and still want to see total bets, we only want to show that they arent connected if they are trying to view "My Bets"

File: .env.development [hunk 2]
```diff
@@ -10,4 +10,4 @@ REACT_APP_ALCHEMY_API_KEY=<ALCHEMY_API_KEY>
 REACT_APP_REGISTRY_CONTRACT="0xCFa36F3692b19FF9472aEc18f7dcf5EB0A29A633"
 REACT_APP_SCANNER_URL=https://goerli.etherscan.io/tx
 REACT_APP_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/horse-link/hl-protocol-goerli
-REACT_APP_MARKET_ORACLE_CONTRACT="0x5b559E0E44aeB2bccfd82CACF9d877EbDd28F116"
+REACT_APP_MARKET_ORACLE_CONTRACT="0x5b559E0E44aeB2bccfd82CACF9d877EbDd28F116"
```
Comment (OptimusOpus) [inline .env.development:13]: the new REACT_APP_MARKET_ORACLE_CONTRACT is 0x0e7983965d231b98804AFbEF134b73c6CF56e8Ec

Comment (OptimusOpus) [reply .env.development:13]: nvm, lets swap that over in its own PR

**Review by OptimusOpus — DISMISSED** [review]

**Review by pondpiu — APPROVED** [review]

**Review by xiangzhe0812 — APPROVED** [review]

**Pull request merged**